src/main/java/org/mule/extension/db/internal/domain/connection/oracle/OracleConnectionUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.extension.db.internal.domain.connection.oracle;
+
+/**
+ * Utils for the oracle db connection
+ *
+ * @since 1.5.2
+ */
+public class OracleConnectionUtils {
+
+  private OracleConnectionUtils() {
+  }
+
+  public static String getOwnerFrom(String typeName) {
+    if (!typeName.contains(".")) {
+      return null;
```

Comment:
what's this method for? Also do not return null. Return Optional.empty()

Reply:
Because in oracle there can be different data types with different owners, but the same name. And the user can use the data type with its owner or just with the name.